content/automate/ManageIQ/AutomationManagement/AutomationManager/Provisioning/Profile.class/__class__.yaml

```diff
@@ -0,0 +1,33 @@
+---
+object_type: class
+version: 1.0
+object:
+  attributes:
+    description: Profile
+    display_name:
+    name: Profile
+    type:
+    inherits:
+    visibility:
+    owner:
+  schema:
+  - field:
+      aetype: attribute
+      name: state_machine
```

Comment:
This whole file could go away if we don't care about per ldap group statemachines...